### ISSUE TITLE
 Is the LM better than MLM?

### ISSUE THREAD

**Issue by SKBL5694** [salesforce/BLIP#100]
I found in the paper BLIP, you use define the loss  as ITC + ITM + LM. However, in ALBEF, the loss is defined as ITC + ITM +MLM. Is the LM better than MLM or or there are other reasons you used LM instead of MLM?

**Comment by LiJunnan1992** [comment]
Hi, the primary reason for using LM is because we want to enable image-to-text generation capability. Both losses perform similarly in terms of VL representation learning (MLM can be slightly better sometimes).

**Comment by LiJunnan1992** [comment]
For BLIP, the decoder in the pre-trained model. For ALBEF, we use the pre-trained encoder model to initialize the decoder.
